packages/react-components/src/organisms/InterestGroupCard.tsx

```diff
@@ -53,12 +59,19 @@ const InterestGroupCard: React.FC<InterestGroupCardProps> = ({
       </LinkHeadline>
       {!active && <StateTag icon={inactiveBadgeIcon} label="Inactive" />}
     </div>
+    {googleDrive && (
+      <Link href={googleDrive} buttonStyle small>
+        {googleDriveIcon} Access Drive
+      </Link>
```

Comment:
It seems the background is transparent instead of white
![Screenshot 2024-02-14 at 16 36 38](https://github.com/yldio/asap-hub/assets/16595804/a9f4e368-ce20-4ada-8716-546200670180)